relay-event-schema/src/protocol/device_class.rs

```diff
@@ -200,6 +200,80 @@ fn model_to_class(model: &str) -> Option<DeviceClass> {
         "iPad16,5" => Some(DeviceClass::HIGH),
         "iPad16,6" => Some(DeviceClass::HIGH),
 
-        _ => None,
+        // If we don't know the model it's a new device and therefore must be high.
+        _ => Some(DeviceClass::HIGH),
```

Comment:
I assume our list of 'old' devices is extensive enough?

Reply:
Yes, indeed. It's extensive. Thanks for double checking 🙏